client/src/components/homePage.jsx

```diff
@@ -1,13 +1,54 @@
 import React from 'react';
+import DogCard from './card/dogCard.jsx';
 
-import { Typography } from '@mui/material';
+import {
+  Button,
+  Grid,
+  TextField,
+  Typography,
+} from '@mui/material';
+import SearchIcon from '@mui/icons-material/Search';
+
+const searchBar = () => {
```

Comment:
might make sense to make the search bar its own component in a separate file especially once searching functionality has to be added